articles/openshift/toc.yml

```diff
@@ -76,7 +76,7 @@
 
 - name: Resources
   items:
-    - name: Common questions
+    - name: FAQ
```

Comment:
I think we need to change this to "Frequently asked questions:

Reply:
Got it.